src/hooks/use-content-tabs.ts

```diff
@@ -20,27 +20,36 @@ export function useContentTabs() {
 				title: __( 'Share' ),
 			},
 			{
-				order: 3,
+				order: 4,
 				name: 'import-export',
 				title: __( 'Import / Export' ),
 			},
 			{
-				order: 4,
+				order: 5,
 				name: 'settings',
 				title: __( 'Settings' ),
 			},
 		];
 
+		if ( siteSyncEnabled ) {
+			tabs.push( {
+				order: 3,
```

Comment:
Let's change the order of Share and Sync. The Sync tab should be the second one, following the RToz6tIuQ7nlZrikBte4GU-fi-4491_115759.

![image](https://github.com/user-attachments/assets/f3b8b055-f876-473e-97f1-8216356f3e30)


```suggestion
				order: 2,
```

Reply:
Sounds good 👍 I looked at a different design first where it did not have the `Share` tab at all. I will make the adjustments!

Reply:
Updated in https://github.com/Automattic/studio/pull/590/commits/78852f719c7f1c82cb497979310a4899e145b011